packages/elements/src/internals/machines/sign-in/router.machine.ts

```diff
@@ -87,9 +87,10 @@ export const SignInRouterMachine = setup({
 
       const session = id || createdSessionId || lastActiveSessionId || null;
 
-      const beforeEmit = () =>
-        context.router?.push(context.router?.searchParams().get('redirect_url') || context.clerk.buildAfterSignInUrl());
-      void context.clerk.setActive({ session, beforeEmit });
+      void context.clerk.setActive({
+        session,
+        redirectUrl: context.router?.searchParams().get('redirect_url') || context.clerk.buildAfterSignInUrl(),
```

Comment:
@BRKalow would this change the behavior of the AIO components? My understanding is that `Clerk#navigate` will use the `routerPush`/`routerReplace` functions passed to `ClerkProvider` so it should be okay, just want to verify that this is true for the AIO components as well.

Reply:
It should be okay for this case 👍 will test